govc/library/item/upload.go

```diff
@@ -63,8 +64,8 @@ func (cmd *upload) Description() string {
 	return `List library items.
 
 Examples:
-  govc library.item.upload library_name item_name
-  govc library.item.upload library_name item_name -json | jq .`
+  govc library.item.upload library_name file_name
```

Comment:
Hi @dougm,

I believe `item_name` is correct here as you need to know both the name of the library and item to which the file(s) is/are being uploaded.

Reply:
But this command creates the item for you, from the local file_name.

Reply:
Hi @dougm,

Yep. I wish I'd realized that sooner :)